airgun/views/all_hosts.py

```diff
@@ -59,20 +65,25 @@ class AllHostsTableView(BaseLoggedInView, SearchableViewMixinPF4):
     )
     bulk_actions = AllHostsMenu()
     bulk_actions_kebab = Button(locator='.//button[@aria-label="plain kebab"]')
-    bulk_actions_menu = Menu(locator='.//div[@data-ouia-component-id="hosts-index-actions-kebab"]')
+    bulk_actions_menu = PF5Menu(
+        locator='.//div[@data-ouia-component-id="hosts-index-actions-kebab"]'
+    )
+    bulk_actions_manage_content_menu = PF5Menu(
+        locator='//li[contains(@class, "pf-v5-c-menu__list-item")]//button[span/span[text()="Manage content"]]/following-sibling::div[contains(@class, "pf-v5-c-menu")]'
+    )
 
     table_loading = Text("//h5[normalize-space(.)='Loading']")
     no_results = Text("//h5[normalize-space(.)='No Results']")
-    manage_columns = Button("Manage columns")
-    table = PatternflyTable(
+    manage_columns = PF5Button("Manage columns")
```

Comment:
```suggestion
    manage_columns = PF5Button('Manage columns')
```
Making string format consistent

Reply:
As you can see, I did not fiddle with the string here and the discrepancy between the 'String' and "String" usage within the airgun is almost everywhere. I can change it, that's no problem, but I wanted to point out that this issue is everywhere in airgun :/ Which should not be an excuse I agree,

Reply:
@vijaysawant I suggest I do a separate PR sometime in the future that will unite the string designation in Airgun

Reply:
Okay, please make a github issue or keep in your todo list.
Thanks for quick action.

Reply:
https://github.com/SatelliteQE/airgun/issues/1794